rn/release-information/release-notes-21-04-update1.adoc

```diff
@@ -45,3 +49,24 @@ The following table outlines the release particulars:
 * Fixes an issue with filesystem monitoring on pods that use NFS volumes with root squashing enabled. With this fix, NFS volumes with root squashing enabled are skipped to avoid errors.
 //#27983
 * Fixes an issue with the missing content-encoding header when WAAS is enabled.
+
+
+=== Known issues
+
+// #26088
+* Defender containers in Defender DaemonSets still display as vulnerable after upgrade.
+This happens because the auto-upgrade mechanism updates the Defender binary only, not the entire image, and the vulnerabilities exist in the image base layer.
+To resolve the issue, redeploy all Defender DaemonSets.
+
+
+=== Upcoming deprecations
+
+Starting on June 14, 2021, we'll publish https://prisma.pan.dev/api/cloud/cwpp[API documentation] for our stable endpoints only.
```

Comment:
Is there a reason we add a separate line with the link for the stable endpoints instead of making an inline link [stable endpoints](https://prisma.pan.dev/docs/cloud/cwpp/stable-endpoints)

Reply:
Good point. Fixing this now.